src/nginxconfig/templates/callouts/contribute.vue

```diff
@@ -61,10 +64,12 @@ THE SOFTWARE.
                 return this.$data.scrolled && !this.$data.closed;
             },
             style() {
-                return this.visible ? undefined : {
+                const nonVisibleStyle = {
                     opacity: 0,
                     pointerEvents: 'none',
                 };
+
+                return this.visible ? undefined : nonVisibleStyle;
```

Comment:
@MattIPv4 I have extracted the value to a constant because `eslint` and `prettier` style for indentation conflicted with each other. Extracting it to a variable solves the issue.

This is prettier error
<img width="584" alt="image" src="https://github.com/digitalocean/nginxconfig.io/assets/26411488/c9eb0f91-a14e-407f-b94c-b2654164cea9">

This is eslint error 
<img width="584" alt="image" src="https://github.com/digitalocean/nginxconfig.io/assets/26411488/f4ee9944-0c0f-425e-b849-9b33f869de40">

There is an open issue about this. https://github.com/prettier/prettier/issues/4203

Reply:
@MattIPv4 Please resolve the conversation if it's okay.